javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javassistmodel/JavassistConstructorDeclaration.java

```diff
@@ -123,7 +123,7 @@ public ResolvedParameterDeclaration getParam(int i) {
     }
 
     @Override
-    public AccessSpecifier accessSpecifier() {
+    public Modifier.Keyword accessSpecifier() {
```

Comment:
Mmm, AccessSpecifier includes the default package access for which there is no corresponding modifier

Reply:
But there *is* a corresponding `Modifier.Keyword`.